week3_model_free/experience_replay.ipynb

```diff
@@ -212,7 +212,7 @@
     "    - train agent using agent.update(...) whenever possible\n",
     "    - return total reward\n",
     "    :param replay: ReplayBuffer where agent can store and sample (s,a,r,s',done) tuples.\n",
-    "        If None, do not use experience replay\n",
+    "        If None, do not use an experience replay\n",
```

Comment:
I think it's fine not to include any article here. "Experience replay" here is an uncountable entity — we don't imply that there are multiple "replays" possible, for example.